#
# Message for the SPCU sensor readings
# Sensors can be configures diffrently
#
Header    header         # header
int16[8]  sensor         # Sensor
#int16   sensor1        # Actuator 1
#int16   sensor2        # Actuator 2
#int16   sensor3        # Actuator 2
#int16   sensor4        # Actuator 3
#int16   sensor5        # Actuator 3
#int16   sensor6        # Actuator 4
#int16   sensor7        # Actuator 4

